foursight_core/app_utils.py

```diff
@@ -452,26 +502,49 @@ def get_jwt_token(self, request_dict) -> str:
                 cookie_split = cookie.strip().split('=')
                 if len(cookie_split) == 2:
                     cookie_dict[cookie_split[0]] = cookie_split[1]
-        token = cookie_dict.get('jwtToken', None)
+        token = cookie_dict.get('c4_st', None)
```

Comment:
I see you using `c4_st` in various places. Should this be a named constant?

Reply:
I will add this to utils